components/checkbox/style.css

```diff
@@ -37,6 +37,11 @@
   background-color: #003366;
 }
 
+/* Custom checkbox has blue outline when in focus */
+.checkbox input:focus ~ .checkmark {
+  outline: rgb(59, 153, 252) solid 5px;
```

Comment:
I think we're using hex values in other CSS. Might be worth using them here for consistency.